config/development.yaml

```diff
@@ -7,10 +7,12 @@ persistence:
       cassandra:
         hosts: "127.0.0.1"
         keyspace: "cadence"
+        protoVersion: 4
```

Comment:
Please also update `config_template.yaml` in the `docker` folder.

Reply:
Done